readme.md

```diff
@@ -124,6 +124,72 @@ Also, you may be interested in adjusting Nextcloud's Datadir to store the files
 
 ⚠️ **Please note:** Almost all commands in this project's documentation use `sudo docker ...`. Since `sudo` is not available on Windows, you simply remove `sudo` from the commands and they should work.  
 
+
+### Synology DSM Setup
+<details>
+    <summary>For a AIO setup in Docker on a Synology NAS, there are a few adjustments necessary to make it run well, especially when running the integrated reverse proxy or firewall.</summary>
+    
+    
+The main issue is that installing AIO through the **Docker GUI doesn't work**, so you need to either use SSH or create a user-defined script task in the task scheduler as the user 'root' and run it.
+Before you start make sure to create two folders on you NAS `/volume1/docker/nextcloud/data` `/volume1/docker/nextcloud/backup`
+
+The compose script without reverse proxy
+```
+sudo docker run \
+--sig-proxy=false \
+--name nextcloud-aio-mastercontainer \
+--restart always \
+--publish 80:80 \
+--publish 8080:8080 \
+--publish 8443:8443 \
+--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
+-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data" \
+-e DOCKER_SOCKET_PATH="/volume1/docker/docker.sock" \
+nextcloud/all-in-one:latest 
+```
+
+ <details>
+ <summary>Setup with reverse proxy </summary>
+
+The compose script with reverse proxy which points to `localhost` at port 11000 with **HTTP** internally
+
+```
+sudo docker run \
+--sig-proxy=false \
+--name nextcloud-aio-mastercontainer \
+--restart always \
+--publish 8080:8080 \
+-e APACHE_PORT=11000 \
+-e APACHE_IP_BINDING=127.0.0.1 \
+--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+--volume /volume1/docker/docker.sock:/var/run/docker.sock:ro \
+-e NEXTCLOUD_DATADIR="/volume1/docker/nextcloud/data" \
+-e DOCKER_SOCKET_PATH="/volume1/docker/docker.sock" \
+nextcloud/all-in-one:latest
+```
+
+![Screenshot 2023-01-20 at 11 54 41](https://user-images.githubusercontent.com/70434961/213679766-96c44fd8-12e7-4f81-9227-48476c58aaef.jpg)
+![Screenshot 2023-01-20 at 11 55 06](https://user-images.githubusercontent.com/70434961/213680608-0e11c6dc-0fa2-4961-9f55-4f7f8c977e69.jpg)
```

Comment:
Lets move this to https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

```suggestion
```

Reply:
Done with https://github.com/nextcloud/all-in-one/pull/1813/commits/1e933e0b053991d0c8aa2532c6f90a20570d0840